sig-network/role-sig-chair.md

```diff
@@ -0,0 +1,67 @@
+# Role: SIG Chair
+
+## Background
+
+Please see the
+[standard description of a SIG chair](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md#chair),
+which does a good job describing the expectations of the more “administrative”
+aspect of being a SIG chair.
+
+# What’s expected
+
+Rather than repeat the doc linked above, this doc will just add some color.
+
+The minimum effective engagement for a SIG chair is to “run the show” (really
+“make sure the show is running” - delegation is good!).  That generally means
+building an agenda for and running regular meetings, curating our
+community-facing metadata, and acting as liaison to other groups (other SIGs
+and steering, mostly).
+
+This includes new-issue triage - either doing the triage or leading it (e.g. in
+the regular SIG meetings).
+
+# Above and beyond
+
+The basic expectations above should not be a huge time commitment.  SIG chairs
+who want to have more impact and can commit more time can engage in a multitude
+of additional ways.  While this can include being an active code contributor or
+SIG TL, an effective SIG chair does not NEED to be a code contributor.
+
+Some concrete things that SIGs need to do, which the chair-people can help with
+include:
```

Comment:
Commenting as this was brought up as a possible thing we could adopt across all SIGs at the Jan. 2023 Leads meetings.

Reading through this doc initially, I didn't catch that the bulleted list that follows is intended as the responsibilities of the *SIG*, since the doc is geared towards the chair. (thanks @marosset for that catch!) My initial reaction was that this list of expectations for the SIG chair is simply far too much for a single person to be expected to do.

I would suggest that for clarity, this doc be reframed as "these are the things that need to happen in the SIG" rather than as expectations of the chairs.

It may also be relevant to share the leads curriculum I wrote back in 1.22, which includes areas of responsibility for SIG Leads as well as homework to better understand the various parts of SIGs: https://github.com/kubernetes/community/blob/master/mentoring/programs/leads-curriculum.md

Reply:
_optional_ change

To avoid confusing readers:
```suggestion
There's a longer list of concrete things that SIGs need to do, and which the chair-people can
help. As a chair, these are **not** your responsibility, but the chair is accountable for
taking steps to help make things happen.

Examples of these activities include:
```
?
Does that help?

Reply:
The intention was really "these are things that most/all SIGs do (or should), any or all which an engaged sig-chair can facilitate or even do themselves.

Reply:
I just don't think it's realistic for an individual SIG Chair to be attempting to do all of the above by themselves. It feels a little bit like writing aspirational responsibilities for a job and then being surprised that people don't apply because they can't imagine themselves doing more than 50% of it.

By putting this in a "SIG Chair" doc rather than "SIG Activities" doc, this comes across to me as a list of activities we expect an individual to be doing rather than the group of SIG volunteers, even though we have the brief caveat written out. I don't think that framing is sustainable, and so in its current form I wouldn't support rolling this out as requirements for all chairs.

I would much rather see us writing documentation that encourages us to build SIGs where responsibilities and recognition are both widely shared amongst a large pool of volunteers vs. setting up more single points of failure and "heroes."

Reply:
Not "all".

The previous text said "can help with" and "can engage in a multitude of additional ways".

I have revised it to say "Some concrete things that SIGs need to do, which the chair-people can facilitate, participate in, or even do themselves include:"

I don't think I am asking for or implying heroics, am I?  I'm just trying to say "here's a bunch of things you can do it you want to engage more"

Reply:
@ehashman I think many of these activities are already a part of the chair definition. This writeup just expands on those. What I feel happens here is that this document talks about bare minimum chair responsibilities when everything that CAN be delegated IS delegated. In situations when it's true, not delegating some of these items is expanded in above and beyond section.

Knowing that not everything will be delegated, seeing this list may be scary. Suggestion to change it to "SIG Activities" may be a reflection of the fact that delegation today is not trivial with lack of time and energy in SIGs. Calling these responsibilities "SIG Activities"  shifts the focus from "chair have to actively delegate/do things" -> "common list of tasks that everybody have to contribute to". It also helps to highlight the fact that all of the above MUST happen.

I believe that actively delegating is an expectation from a chair. So if we are keeping this document structure, I'd suggest we highlight this as a primary responsibility. 

I also suggested more changes here: https://github.com/kubernetes/community/pull/7034#issuecomment-1378459260